package_bag/tests.py

```diff
@@ -25,6 +28,49 @@
 RIGHTS_FIXTURE_DIR = join(settings.BASE_DIR, 'fixtures', 'rights')
 
 
+class TestHelpers(TestCase):
+
+    def test_expected_filename(self):
+        passing_filenames = ["7d24b2da347b48fe9e59d8c5d4424235.tar", "4b4334fba43a4cf4940f6c8e6d892f60.tar.gz"]
+        failing_filenames = ["6163b89b00bb4c5cbfc50eb34cb49a75", "example_bag.tar"]
+        for filename in passing_filenames:
+            self.assertTrue(expected_file_name(filename))
+        for filename in failing_filenames:
+            self.assertFalse(expected_file_name(filename))
+
+
+class TestS3Download(TestCase):
+    fixtures = [join(settings.BASE_DIR, "fixtures", "s3_download.json")]
+
+    def configure_uploader(self, upload_list):
+        """Sets up an 3ObjectDownloader with mocked s3 bucket and objects."""
+        object_downloader = S3ObjectDownloader()
+        region_name, access_key, secret_key, bucket = settings.S3
+        s3 = boto3.resource(service_name='s3', region_name=region_name, aws_access_key_id=access_key, aws_secret_access_key=secret_key)
+        s3.create_bucket(Bucket=bucket)
+        s3_client = boto3.client('s3', region_name=region_name)
+        object_downloader.bucket = s3.Bucket(bucket)
+        for item in upload_list:
+            s3_client.put_object(Bucket=bucket, Key=item, Body='')
+        return object_downloader
+
+    @mock_s3
+    def test_get_list_to_download(self):
+        """Test list contains one filename than is in the database; the file that is in the database should not be downloaded"""
```

Comment:
```suggestion
        """Test list contains one filename that is in the database; the file that is in the database should not be downloaded"""
```
Sorry, screwed this up earlier.